build/dependencies.yaml

```diff
@@ -124,7 +124,7 @@ dependencies:
       match: minimum_go_version=go([0-9]+\.[0-9]+)
 
   - name: "k8s.gcr.io/kube-cross: dependents"
-    version: v1.16.6-1
+    version: v1.16.7-1
```

Comment:
`v1.16.7-0` or `v1.16.7-1`?

Reply:
we built `v1.16.7-1` following the other, but I agree that should be `-0`
:/

Reply:
https://github.com/kubernetes/release/pull/2197/files#diff-9d27d65f78671b29185b413d109ff0a70ddc2a5f7e5ca474ce6674224bff1ca1R19

Reply:
> `v1.16.7-0` or `v1.16.7-1`?

`v1.16.7-1` is correct. The `-1` is considered to be a "revision" number, similar to Debian revisions, starting at 1.